### DISCUSSION TITLE
How to turn on the "GPTK's HUD", on a MyCoolWrapper + GPTK?

### DISCUSSION THREAD

**Discussion by ferreira-ff** [The-Wineskin-Project/WineskinServer#434] [Q&A] [answered]
Hello. I have picked up, the latest release of Wineskin 1.8.4 (**WineCX22.1.1-8 + Wineskin-2.9.1.9**).

**MyCoolWrapper DFLT:**
I have made a default "**MyCoolWrapper**"; tested and backed it up.

**MyCoolWrapper DXVK:**
Then; I have added the latest DXVK to the Wrapper. On the prefix folder, user.reg, I have put **"DXVK_HUD"="full"**. Every time I install a game, the **DXVK HUD** is on by default. Tested and backed it up.

**MyCoolWrapper GPTK:**
After that; I have added the latest GPTK to the default Wrapper. It is working perfectly, but I am unable to make the HUD work!
On the prefix folder, user.reg, I have put **"MTL_HUD_ENABLED"="1"**. Unfortunately, I am unable to make the HUD show up. Somehow; I managed to make it work once, but I erased that testing wrapper and I don't remember, what was the right command and if it was, on the user.reg file...
_I have tried almost everything, without success:_
"MTL_HUD_ENABLED"="1"
"MTL_HUD_ENABLED"="Y"
MTL_HUD_ENABLED=1
MTL_HUD_ENABLED=Y

So; my question is: _**what is the right command that I should write**_, to make Apple's GPTK HUD turn on, when installing and playing a game? Some help would be really appreciated.

Kind regards to all

**Comment by ferreira-ff** [comment]
While waiting for help, I have found a "temporary" solution:

_Terminal Command_
open **thegameiwanttoplay.app** --env MTL_HUD_ENABLED=1
(the GPTK HUD works, while the game is running)

If you create a shell scrip, do not forget to make it executable: **chmod +x** file.command

Kind regards to all

**Comment by Gcenx** [comment, marked as answer]
You put these things into the “Unix” field

Here’s an example
```
export MTL_HUD_ENABLED=1; export DXVK_HUD=full
```

**Reply by ferreira-ff** [reply]
Yeah; an answer and from the Master Gcenx himself; thank you very much.
My friend; very well explained, simple and effective. The "HUD" is working perfectly.
Now, I can set the game's configuration according with the HUD information, then remove it and enjoy the game...

I wanted to offer you and the helping team, a coffee. You could have "ko-fi" as Mr. Marovitz. Wineskin & Whisky are simply superb!

Again; thank you for the help. Kind regards to all.
<img width="1280" alt="Captura de ecrã 2023-07-31, às 16 09 45" src="https://github.com/Gcenx/WineskinServer/assets/138707753/4185e2ab-ed76-4578-87b5-a9389dda0b86">

**Reply by Gcenx** [reply]
I’d already had PayPal linked but just setup ko-fi account just now and added.

I should see what else everyone using these days.

**Reply by ferreira-ff** [reply]
Me too. My daughters are almost adults and I am getting old as a tugboat. Kind regards.
